tests/util/test_util.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import pandas as pd
+from climakitae.util.utils import read_csv_file
+from climakitae.core.constants import WARMING_LEVELS
+from climakitae.core.paths import gwl_1850_1900_file, gwl_1981_2010_file
```

Comment:
Can you also add a brief string description at the very top of the file? Like in this test file: https://github.com/cal-adapt/climakitae/blob/afc2dc2618240a85d31cab51c80dd9b7b7b0daf5/tests/data_interface/test_variable_descriptions_csv.py#L1